tests/geometry/test-distance.cpp

```diff
@@ -158,8 +158,8 @@ void test_point_segment_distance()
 void test_point_triangle_distance_2d()
 {
     const geode::Point2D a{ { 0.0, 0.0 } };
-    const geode::Point2D b{ { 1.0, 0.0 } };
-    const geode::Point2D c{ { 1.0, 1.0 } };
+    const geode::Point2D b{ { 1.0, 1.0 } };
+    const geode::Point2D c{ { 1.0, 0.0 } };
```

Comment:
This change is OK for me, but it is a bit strange since you may break other tests based on previous configuration.